### DISCUSSION TITLE
How can Worker memory usage be managed ?

### DISCUSSION THREAD

**Discussion by inverted-capital** [denoland/deno#12637] [Q&A]
In this issue #6916 the ability was added to register a callback when an isolate is under memory pressure, so it can be terminated without bringing down the whole v8 instance.  How can we access this capability ?

Specifically we would like the means of running untrusted code within a Worker instance, taking advantage of the sandboxing provided by Deno, and the efficiencies in resource usage and latency afforded by v8 isolates.  Thank you for this great project.

**Comment by ykahlon** [comment]
I am interested to know whether you got an answer to this question?

**Reply by inverted-capital** [reply]
I did not get an answer yet

**Comment by josephrocca** [comment]
For reference, I've made an issue about this which contains step-by-step instructions to building a custom `deno` binary that has a `WEB_WORKER_HEAP_LIMIT_BYTES` environment variable:
* https://github.com/denoland/deno/issues/26202
